server/routes/polls/add.route.js

```diff
@@ -0,0 +1,37 @@
+const express = require("express");
+const router = express.Router();
+const Poll = require("../../models/polls");
+const upload = require("./utils");
+
+router.post("/", upload.any(), (req, res) => {
+	const errors = [];
+	if (!req.body.title) errors.push({ title: "Invalid title" });
+	if (!req.body.userId) errors.push({ userId: "Invalid user id" });
+	if (req.files.length != 2)
```

Comment:
```suggestion
	if (req.files && req.files.length != 2)
```
because it fails if `req.files` is `undefined`